manifest_staging/charts/secrets-store-csi-driver/values.yaml

```diff
@@ -9,6 +9,15 @@ linux:
   tolerations: []
   metricsAddr: ":8080"
 
+  registrarImage:
```

Comment:
Can we group all the image configurations in `values.yaml` together? `registrarImage` and `livenessProbeImage` section right after `image` section, so it's all in order.